src/uu/wc/src/wc.rs

```diff
@@ -66,57 +65,77 @@ struct Result {
     max_line_length: usize,
 }
 
-static NAME: &str = "wc";
+static ABOUT: &str = "print newline, word, and byte counts for each file.";
```

Comment:
```suggestion
static ABOUT: &str = "Display newline, word, and byte counts for each FILE, and a total line if
more than one FILE is specified.";
```

More `coreutils` standard and GNU similar?